sinopia_indexer/index.js

```diff
@@ -0,0 +1,41 @@
+// Copyright 2019 Stanford University see Apache2.txt for license
+
+const elasticsearch = require('elasticsearch')
+const Stomp = require('stomp-client')
+const superagent = require('superagent')
+
+const searchHost = process.env.SEARCH_HOST || 'localhost'
+const brokerHost = process.env.BROKER_HOST || 'localhost'
```

Comment:
oh cool.  i recall us looking for a way to get at env vars when @jermnelson and i were pairing w/ sarav on the cognito work he did before he left.  can't recall that context well enough to remember whether we could use the same approach there, but prob a useful thing to keep in mind as a tool anyway.

Reply:
Hi @jmartin-sul , the [.dotenv](https://www.npmjs.com/package/dotenv) npm module is the one you where thinking of that Sarav was using.